resources/views/createCourse.blade.php

```diff
@@ -0,0 +1,145 @@
+
+@extends('layouts.app')
+
+@section('title', 'Register a course')
+
+@section('content')
+<div class="padded content">
+	<h1>Register a course</h1>
+	<form class="ui error form" method="POST" action="/courses">
+		@csrf
+		<!-- Course name -->
+		<div class="field">
+			<label for="courseName">Course name</label>
+			<input type="text" name="courseName" id="courseName" required>
+		</div>
+		<!-- Teacher(s) -->
+		<div class="field">
+			<label for="teachers">Teacher(s)</label>
+			<select class="ui fluid search dropdown" name="teachers" id="teachers" multiple required>
+				<option value="1">Mr. Monday</option>
+				<option value="2">Mrs. Tuesday</option>
+				<option value="3">Ms. Wednesday</option>
+				<option value="4">Mr. Thursday</option>
+				<option value="5">Mrs. Friday</option>
+			</select>
+		</div>
+		<!-- Course semester -->
+		<div class="field">
+			<label for="courseSemester">Course semester</label>
+			<input type="text" name="courseSemester" id="courseSemester" placeholder="Spring/Fall/Summer/Winter and year Ex. Fall 2018" required>
+		</div>
+		<!-- Course type -->
+		<div class="field">
+			<label for="courseType">Course type</label>
+			<select class="ui fluid search dropdown" name="courseType" id="courseType" required>
+				<option value=""></option>
+				<option value="1">Client</option>
+				<option value="2">Supplier</option>
+			</select>
+		</div>
+		<!-- Course schedule -->
+		<div class="field">
+			<label for="couseSchedule">Course schedule</label>
+			<select class="ui fluid search dropdown" name="courseType" id="courseSchedule" multiple required>
+				<option value="monday">Monday</option>
+				<option value="tuesday">Tuesday</option>
+				<option value="wednesday">Wednesday</option>
+				<option value="thusday">Thursday</option>
+				<option value="friday">Friday</option>
+				<option value="saturday">Saturday</option>
+			</select>
+		</div>
+		<!-- Course hours -->
+		<div class="field">
+			<label for="courseHours">Starting time</label>
+			<input type="time" name="courseHours" id="courseHours" required>
```

Comment:
`time` inputs [have horrible browser support rn](https://caniuse.com/#feat=input-datetime). I think we should look for a workaround.

Reply:
puede ser un dropdown con cada media hora dentro de horario razonable :) [07:00, 07:30, 08:00...]

Reply:
Creo que un dropdown es lo más sano/sencillo, thanks!